code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm

```diff
@@ -61,6 +61,7 @@
 		for(var/mob/O in viewers(user, 3))
 			if((O.client && !( O.blinded )))
 				to_chat(O, "<span class='notice'>The locker has been [locked ? null : "un"]locked by [user].</span>")
+				playsound(src.loc, 'sound/machines/click.ogg', 15, 1, -3)

```

Comment:
while fox's comment could be less forceful he has a point
you should only play this once outside of the for(var/mob/O) loop not for each mob